list/List.js

```diff
@@ -132,24 +133,47 @@ define([
 		// the SPACE key to (de)select an item.
 		multiCharSearchDuration: 0,
 
-		setAttribute: dcl.superCall(function (sup) {
-			return function (attr, value) {
-				sup.apply(this, arguments);
-				if (attr === "role") {
-					// Only role=grid should have aria-readonly.  It's meaningless for role=listbox or role=menu.
-					if (value === "grid") {
-						this.setAttribute("aria-readonly", "true");
-					} else {
-						this.removeAttribute("aria-readonly");
-					}
+		/**
+		 *	If `true` and the list's store is empty, the NoItemContainer element will be shown.
+		 * @member {boolean}
+		 * @default "false"
```

Comment:
Presumably there shouldn't be quotes around "false" because this prop is a boolean.   (There's also some extra space at 137.)